Palace.xcodeproj/project.pbxproj

```diff
@@ -4874,11 +4874,9 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Palace/PalaceDebug.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "Apple Distribution";
-				CODE_SIGN_STYLE = Manual;
-				CURRENT_PROJECT_VERSION = 202;
-				DEVELOPMENT_TEAM = "";
-				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = 88CBA74T8K;
+				CODE_SIGN_STYLE = Automatic;
```

Comment:
`CODE_SIGN_STYLE` should be manual, with "Apple Distribution" in `CODE_SIGN_IDENTITY`